debian/control

```diff
@@ -520,7 +520,8 @@ Description: MariaDB database client (metapackage depending on the latest versio
 
 Package: mariadb-plugin-connect
 Architecture: any
-Depends: libxml2,
+Depends: libgcj14-awt | libgcj15-awt | libgcj14-awt | libgcj15-awt | libgcj17-awt | openjdk-7-jre-headless | openjdk-8-jdk-headless | openjdk-11-jre | openjdk-17-jre,
+         libxml2,
```

Comment:
Why libxml2 is added? Is dependency of libgcj-awt?

Reply:
Package `libgcj-awt` will pull in it's own dependencies automatically, there is no need to define dependencies of dependencies. Package `libxml2` is not mentioned in the commit message and not added in this commit as it was already there.

Reply:
Sorry.. it slipped through my eye..